### DISCUSSION TITLE
Export Table Headers

### DISCUSSION THREAD

**Discussion by swordfish291** [EvotecIT/PSWriteHTML#344] [Q&A]
Hello,

While exporting the tables to Excel the headers of the tables are missing. The filer location for the tables is on the top. That might be the problem? I was also looking for a way while copying rows to clipboard to copy the headers as well. Is there a way to achieve that?

Thanks in advance.

Abdul

**Comment by PrzemyslawKlys** [comment]
You've got to provide some example. I've not seen export fail.

**Reply by swordfish291** [reply]
Here is the code the exported files.

`$Verbs = Get-Verb

New-HTML -FilePath ./Verbs.HTML {
    New-HTMLSection -HeaderText "PowerShell Verbs" {
        New-HTMLTable -DataTable $Verbs -Filtering -FilteringLocation Top
    }
} `
[pdf (1).pdf](https://github.com/EvotecIT/PSWriteHTML/files/9897771/pdf.1.pdf)

**Comment by swordfish291** [comment]
Hello,

Following up if you had a chance to look into this.

Thanks
Abdul

**Comment by PrzemyslawKlys** [comment]
Yes, headers are missing in both Export to Excel, Copy when FilteringLocation Top is used. Bottom works fine in both cases. Probably a bug or workaround is needed.

**Reply by PrzemyslawKlys** [reply]
https://datatables.net/forums/discussion/comment/214981#Comment_214981 - has a solution - this needs to be applied to sources
